cpp/server/ct-server.cc

```diff
@@ -537,17 +413,18 @@ int main(int argc, char* argv[]) {
     config.set_minimum_serving_fraction(1);
     LOG(INFO) << "Setting default single-node ClusterConfig:\n"
               << config.DebugString();
-    consistent_store.SetClusterConfig(config);
+    server.consistent_store()->SetClusterConfig(config);
 
     // Since we're a single node cluster, we'll settle that we're the
     // master here, so that we can populate the initial STH
     // (StrictConsistentStore won't allow us to do so unless we're master.)
-    election.StartElection();
-    election.WaitToBecomeMaster();
+    server.election()->StartElection();
+    server.election()->WaitToBecomeMaster();
 
     {
       EtcdClient::Response resp;
-      util::SyncTask task(&internal_pool);
+      ThreadPool pool(1);
+      util::SyncTask task(&pool);
```

Comment:
You could just use `event_base` for this, there's really nothing of interest done on the executor of that task (only pinging the `Notification`).